server/anp/photo.py

```diff
@@ -0,0 +1,131 @@
+
+import math
+import requests
+import superdesk
+
+from flask import json, request
+from datetime import datetime
+from xmlrpc.client import ServerProxy
+from superdesk.utils import ListCursor
+from superdesk.utc import local_to_utc
+from superdesk.logging import logger
+
+
+TZ = 'Europe/Amsterdam'
+
+THUMBNAIL = 1
+PREVIEW = 2
+TITLE = 4
+DESCRIPTION = 8
+KEYWORDS = 16
```

Comment:
just a small note, Python has now the enum module (https://docs.python.org/3/library/enum.html) which can be used here and below with `TYPES`